sprints/dashboard/tasks.py

```diff
@@ -71,7 +81,7 @@ def complete_sprints():
 
             issues: List[Issue] = conn.search_issues(
                 **prepare_jql_query_active_sprint_tickets(
-                    list(),  # We don't need any fields here. The `key` attribute will be sufficient.
+                    ['None'],  # We don't need any fields here. The `key` attribute will be sufficient.
```

Comment:
I benchmarked this with iPython and it seems to be significantly increasing the performance (12.6s vs 1.6s). Apparently the Jira lib was doing a `not` check instead of the `is None` here.

Reply:
Thanks!